marimo/_plugins/ui/_impl/input.py

```diff
@@ -125,6 +128,8 @@ class slider(UIElement[Numeric, Numeric]):
     - `stop`: the maximum value of the interval
     - `step`: the slider increment
     - `value`: default value
+    - `debounce`: whether to debounce the slider to only send
+        the value on mouseup
```

Comment:
"whether to debounce the slider to only send the value on mouse-up or drag-end" ?